test/tests/chaincontrols/pid_chain_control/gold/pid_chain_control_out.csv

```diff
@@ -0,0 +1,52 @@
+time,pid
+0,0
+1,2.75
+2,2.0925
+3,2.72457325
+4,2.5335790726461
+5,2.8603066874179
+6,2.8245939220233
+7,3.0347665154606
+8,3.0551757345161
+9,3.210477510423
+10,3.2516113832507
+11,3.3763731064325
+12,3.4240711960446
+13,3.5294172459587
+14,3.5775155875617
+15,3.6692531500248
+16,3.7149091810977
+17,3.7964672179247
+18,3.8383194575783
+19,3.9119700047426
+20,3.949340804225
+21,4.0167598252515
+22,4.0492767273709
+23,4.111829896963
+24,4.1392259316177
+25,4.1981339660993
+26,4.2201260169629
+27,4.2765779276254
+28,4.2927760190116
+29,4.3480245216194
+30,4.3578463789645
+31,4.4133061808244
+32,4.4158794424525
+33,4.4732448152774
+34,4.4672807495646
+35,4.5286794628989
+36,4.5122994252546
+37,4.5805044412438
+38,4.5509940844949
+39,4.629722476961
+40,4.58317830081
+41,4.6775196544384
+42,4.6083363386246
+43,4.7253722621867
+44,4.6254948562475
+45,4.7752000602772
+46,4.6330286760548
+47,4.8295863577061
+48,4.628367040742
+49,4.8920922443363
+50,4.6075489928787
```

Comment:
seen some closer numbers to 5. 
does it make it there eventually?

Reply:
It gets there very slowly - just wanted to show that it's converging. However, I do see that these parameters get there in an oscillatory fashion, so I'm going to change and re-gold.